### PR TITLE
Feat/account rename

### PR DESCRIPTION
**Pull request by TheLeoB**
## Description (optional)
Changing all mentions of 'Private Key Wallet' to 'DeFi Wallet' to maintain consistency with mobile superapp launch. 

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

**Review by jjBlockchain — DISMISSED** [review]

File: packages/blockchain-wallet-v4/src/redux/wallet/sagas.ts [hunk 1]
```diff
@@ -346,6 +346,15 @@ export default ({ api, networks }) => {
           const newLabel = `${BTC_ACCT_NAME}${labelSuffix}`
           yield put(A.wallet.setAccountLabel(i, newLabel))
         }
+        // separate check for 'Private Key Wallet' label
+        // since we check for the label suffix at a later
+        // character
+        if (startsWith(legacyWalletNameV2, acctLabel)) {
+          // pluck label suffix e.g. " 2"
+          const labelSuffix = last(splitAt(18, acctLabel))
```
Comment (dkremniov-bc) [inline packages/blockchain-wallet-v4/src/redux/wallet/sagas.ts:354]:
```suggestion
          const labelSuffix = last(splitAt(legacyWalletNameV2.length, acctLabel))
```

**Review by milan-bc — DISMISSED** [review]

**Review by dkremniov-bc — DISMISSED** [review]

**Review by Dezzj — APPROVED** [review]
Approved

**Review by Dezzj — APPROVED** [review]
Approved

File: packages/blockchain-wallet-v4/data/wallet.v3.json
Comment (TheLeoB) [inline packages/blockchain-wallet-v4/data/wallet.v3.json:1]: fixed this typo

File: packages/blockchain-wallet-v4/data/wallet.v4.json
Comment (TheLeoB) [inline packages/blockchain-wallet-v4/data/wallet.v4.json:1]: fixed this capitalization issue

**Review by jjBlockchain — APPROVED** [review]

**Pull request merged**